### PR TITLE
passing kibana index template metadata

### PR DESCRIPTION
**Pull request by clutroth**

### CODE REVIEW

File: core/src/test/scala/tech/beshu/ror/unit/acl/blocks/rules/KibanaTemplateIndexRuleTests.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import tech.beshu.ror.unit.acl.blocks.rules.utils.IndexNameRuntimeResolvableVariable
+import tech.beshu.ror.utils.TestsUtils._
+
+class KibanaTemplateIndexRuleTests
+  extends WordSpec
+    with MockFactory {
+
+  import org.scalatest.Matchers._
+
+  "set kibana template index if can be resolved" in {
```
Comment (coutoPL) [inline core/src/test/scala/tech/beshu/ror/unit/acl/blocks/rules/KibanaTemplateIndexRuleTests.scala]: what about a test when cannot be resolved?

File: core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala [hunk 2]
```diff
@@ -251,6 +251,7 @@ class RorInstance private(boot: ReadonlyRest,
   private val currentEngine = Atomic(Option(initialEngine))
 
   def engine: Option[Engine] = currentEngine.get().map(_._1)
+  def config: Option[RawRorConfig] = currentEngine.get().map(_._2)
```
Comment (coutoPL) [inline core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala]: do we need it?

**Comment by coutoPL** [comment]
please fill in a description

File: core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala (hunk 2 above)
Comment (clutroth) [reply core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala]: nope

**Comment by clutroth** [comment]
> please fill in a description

I intentionally omitted description. It's not feature by itself. It's support for feature implemented in kibana plugin. I'm not sure if it deserves own description, and mention in changelog.

**Pull request merged**